ui/app/templates/components/job-page/parts/title.hbs

```diff
@@ -23,7 +23,12 @@
         @confirmText="Yes, Stop Job"
         @confirmationMessage="Are you sure you want to stop this job?"
         @awaitingConfirmation={{this.stopJob.isRunning}}
-        @onConfirm={{perform this.stopJob}} />
+        @onConfirm={{perform this.stopJob}}
+        {{keyboard-shortcut 
+          label="Exec"
```

Comment:
Are these `label="Exec"` copy-paste errors?

Reply:
Yes! Thank you